runtime/doc/options.txt

```diff
@@ -2417,6 +2417,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 			   *'ed'* *'edcompatible'* *'noed'* *'noedcompatible'*
 'edcompatible' 'ed'	Removed. {Nvim}
 
+	Support for Ed-compatible mode has been removed.  "set noed" is
+	ignored, and "set ed" is an error.
```

Comment:
I went for consistency with `'compatible'` here as well. Just remove the first sentence entirely?